toolkits/nature/packages/nature-footer/package.json

```diff
@@ -1,9 +1,9 @@
 {
     "name": "@springernature/nature-footer",
-    "version": "2.1.1",
+    "version": "4.0.3",
```

Comment:
Why is this changing here? All the other changes are to the `global-corporate-footer`, this is why travis is complaining.

Reply:
Looks like this needs to stay at `2.1.1` and the `global-corporate-footer` needs to be updated to `4.0.3`

Reply:
Ah-hah!

Reply:
now I've made these changes - I don't think it's 'right' for me to review it (the initial task). @alexkilgour / @jpw is this something you could take a look at for @amyhupe 😄